projects/fsastorefrontstyles/scss/components/cms/_enriched-responsive-banner.scss

```diff
@@ -2,9 +2,68 @@
   .enriched-banner {
     position: relative;
     width: 100%;
+    &-text {
+      background-color: rgba(255, 255, 255, 0.7);
+      position: absolute;
+    }
+    &-title {
+      font-size: var(--cx-h1-font-size);
+      color: var(--cx-color-secondary);
+      font-weight: var(--cx-font-weight-semi);
+      @include media-breakpoint-down(md) {
+        font-size: 4vw;
+      }
+    }
+    &-details {
+      display: block;
+      margin-top: 1vw;
+      pointer-events: none;
+      font-size: var(--cx-h5-font-size);
+      @include media-breakpoint-down(sm) {
+        font-size: 3.5vw;
+      }
+    }
   }
   /* needed as a fix for Enriched banner not displaying correctly SOMETIMES, when resizing window */
   cx-media img {
     width: 100%;
   }
+  .box-size {
```

Comment:
Just `.box`.
I would suggest for start not to have width defined, it should be defined by it's content. We can break content with <br> in impex in two rows (if it is needed at all).

Reply:
And what if customer has longer text, he should also add <br> on the multiple places? With `max-width` we restrict width at some point, I don't see why we shoudn't define it?

Reply:
I see your point, and you are right, but again in your code you have to adjust for different screen sizes. We should keep this basic code - basic. I was thinking that someone may break their titles in impexes if they don't like how the look OOTB. But, maybe we can try to create default box width again with margins through config. That would also be a great showcase how this feature is suppose to work. We can set `calc()` for margin as well so it 'pushes' the box to some width and it looks consistent on all breakpoints.